.kokoro/gas/trigger-protobuf.cfg

```diff
@@ -0,0 +1,50 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build logs will be here
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
+
+# Download Ruby-cloud resources
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/google-cloud-ruby"
+
+# Download trampoline resources
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Download artifacts from placer
+gfile_resources: "/placer/prod/TODO"
```

Comment:
Needs to be specified.

Reply:
/placer/prod/home/protobuf-build/release

Reply:
Thanks!